### PR TITLE
Support custom text color in issuance wizard headers

### PR DESCRIPTION
**Pull request by sietseringers**

### CODE REVIEW

File: descriptions.go [hunk 1]
```diff
@@ -362,6 +363,9 @@ func (wizard *IssueWizard) Validate(conf *Configuration) error {
 	if wizard.Color != nil && !regexp.MustCompile("^#[0-9A-F]{6}$").MatchString(*wizard.Color) {
 		return errors.New("invalid wizard color: must be of the form #RRGGBB")
 	}
+	if wizard.TextColor != nil && !regexp.MustCompile("^#[0-9A-F]{6}$").MatchString(*wizard.TextColor) {
+		return errors.New("invalid wizard text color: must be of the form #RRGGBB")
+	}
```
Comment (ivard) [inline descriptions.go:368]: This double negation confuses me a bit. The `^` tag in the regex also specifies a negation right? So this error should be thrown if it does not match the no-match?

**Review by ivard — COMMENTED** [review]
The change looks good to me. I was only a bit confused by the negation in the regex expression. Is this one fully correct?

File: descriptions.go (hunk 1 above)
Comment (sietseringers) [reply descriptions.go:368]: No, `^` in a regexp is not an inversion; instead it matches only the beginning of the string.

Comment (ivard) [reply descriptions.go:368]: Ah right, you're indeed right. I must have misread the docs there then.

**Review by ivard — APPROVED** [review]

File: descriptions.go (hunk 1 above)
Comment (sietseringers) [reply descriptions.go:368]: You were perhaps confused with when `^` occurs within square brackets, i.e. a [character class](https://golang.org/pkg/regexp/syntax/#hdr-Syntax). In that case it does act as an inversion.

**Pull request merged**